circuits/cpp/barretenberg/ts/src/main.ts

```diff
@@ -140,12 +143,11 @@ export async function contract(outputPath: string, vkPath: string) {
   try {
     await api.acirLoadVerificationKey(acirComposer, new RawBuffer(readFileSync(vkPath)));
     const contract = await api.acirGetSolidityVerifier(acirComposer);
-    if (outputPath === '-') {
-      console.log(contract);
-    } else {
-      writeFileSync(outputPath, contract);
-      console.log(`contract written to: ${outputPath}`);
-    }
+    
+    process.stdout.write(contract);
+    writeFileSync(outputPath, contract);
+
+    debug(`contract written to: ${outputPath}`);
```

Comment:
All input which is just "information" is printed to debug which should print to stderr, instead of console.log which is printing to stdout